### PR TITLE
Add spanish translation for Schnorr Signatures Podcast #2

### PR DESCRIPTION
**Pull request by BlueeeMoon**

### CODE REVIEW

File: andreas-antonopoulos/2018-10-07-andreas-antonopoulos-schnorr-signatures.es.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: Andreas Antonopoulos - Firmas Schnorr (2018-10-07)
+transcript_by: Michael Folkson
```
Comment (grunch) [inline andreas-antonopoulos/2018-10-07-andreas-antonopoulos-schnorr-signatures.es.md:3]: You forget `translation_by` you, you can add it but that is not mandatory

**Review by grunch — COMMENTED** [review]
ACK

great job 😄

**Comment by adamjonas** [comment]
If you'd like to add the `translation_by`, that'd be great, but this is OK to merge as is.

Thanks! 

(Note that this is built on #14)

File: andreas-antonopoulos/2018-10-07-andreas-antonopoulos-schnorr-signatures.es.md (hunk 1 above)
Comment (BlueeeMoon) [reply andreas-antonopoulos/2018-10-07-andreas-antonopoulos-schnorr-signatures.es.md:3]: Thank you, I have added **translation_by**

**Comment by BlueeeMoon** [comment]
> ACK
> 
> great job 😄

Thanks!!

**Comment by BlueeeMoon** [comment]
> If you'd like to add the `translation_by`, that'd be great, but this is OK to merge as is.
> 
> Thanks!
> 
> (Note that this is built on #14)

**Comment by adamjonas** [comment]
@BlueeeMoon did you mean to close this?

**Pull request merged**